scripts/file_to_c.py

```diff
@@ -3,6 +3,8 @@
 #
 # Copyright (c) 2018, Linaro Limited
 #
+from array import array
+
```

Comment:
Why not `#!/bin/env python3`?

Reply:
I guess there are pros and cons with that. If the script work with both Python 2 and 3, then `#!/usr/bin/env python` is better, since it'll work with whatever Python environment that is default on the users machine. However, since we intentionally are aiming for moving all our Python scripts to Python 3 (due to the Python2 deprecation), I think I'm with you @jforissier , better to use what you've proposed.

Reply:
Thanks! Agreed :-) I'll fix it now! Should I create a new commit or just amend this one?

Reply:
> Thanks! Agreed :-) I'll fix it now! Should I create a new commit or just amend this one?

We usually just add a new "fixup" commit to the branch, then later on when where all happy, then we squash those into a single (or more relevant) commits.